addons/maptools/CfgVehicles.hpp

```diff
@@ -43,17 +43,150 @@ class CfgVehicles {
                     exceptions[] = {"isNotDragging", "notOnMap", "isNotInside", "isNotSitting"};
                     showDisabled = 1;
                 };
-                class ACE_MapToolsAlignNorth {
-                    displayName = CSTRING(MapToolsAlignNorth);
+                class ACE_MapToolsAlign {
+                    displayName = CSTRING(AlignTo);
                     condition = QUOTE(GVAR(mapTool_Shown) != 0);
-                    statement = QUOTE(GVAR(mapTool_angle) = 0;);
+                    statement = "";
+                    exceptions[] = {"isNotDragging", "notOnMap", "isNotInside", "isNotSitting"};
+                    showDisabled = 0;
+                    class ACE_MapToolsAlignToPlottingBoardRuler {
+                        displayName = CSTRING(ToPlottingBoardRulerLabel);
+                        condition = QUOTE(GVAR(mapTool_Shown != 0) && GVAR(plottingBoard_Shown == 2));
```

Comment:
```suggestion
                        condition = QUOTE(GVAR(mapTool_Shown) != 0 && GVAR(plottingBoard_Shown) == 2);
```

Reply:
All of the similar changes I have suggested have been reverted due to the force pushes. These are necessary for this PR to be able to run properly.